governance/votes/Executive Vote - November 15, 2023.md

```diff
@@ -0,0 +1,237 @@
+```
+
+#### Approve New Galaxy Payment Instructions
+
+*IPFS Link*: https://gateway.pinata.cloud/ipfs/QmSbwqULr66CiCvNips93vwTrvoTe4i2rJVmho7QfmyqZG
+
+```
+MAKERDAO RESOLUTION
+
+Date: Dated as of related Executive Vote on MakerDAO
+Purpose: Approval of New Payment Instructions to Galaxy Digital Trading Cayman LLC
+
+Reference is made to:
+1. The Amended and Restated Articles of Association (the “Articles”) of the RWA Foundation (“RWAF”), dated July 13, 2022, pursuant to which RWAF’s operations are governed.
+2. The Escrow Agreement dated as of July 12, 2022 (the “Escrow Agreement”), between RWAF, as trust sponsor (the “Trust Sponsor”), RWA Master Participation Trust and Wilmington Savings Fund Society, FSB, as escrow agent (the “Escrow Agent”).
+
+Capitalized terms used herein and not otherwise defined shall have the meaning given to them in the Articles and the Escrow Agreement.
+
+Whereas:
+3. Clause 4.16 of the Articles mandates the directors of the Trust Sponsor to implement all DAO Resolutions (subject to applicable law); and
+4. Clause 4a. of the Escrow Agreement, provides that the Trust Sponsor shall deliver Instructions to the Escrow Agent instructing the Escrow Agent to distribute funds from the Escrow Account as specified in such Instructions and the Trust Sponsor may make and the Escrow Agent shall accept (1) an amendment to existing Instructions or (2) replacement Instructions; provided that, in each case, such amendments or replacement Instructions shall require signature by the Verification Agent confirming its verification of the Affirmative MakerDAO Vote approving such amendment or replacement Instructions.
+
+Authorizations:
+
+In accordance with the governance protocols of MakerDAO on the date specified above and pursuant to clause 4.16 of the Articles, this DAO Resolution approves and ratifies the following actions:
+5. Subject to the approval of the Verification Agent, the director of the Trust Sponsor is instructed to promptly provide new outgoing instructions to the Escrow Agent, as outlined below (“New Instructions”):
+
+Receiving Bank Routing No: 021000018
+ABA Swift IRVTUS3N
+Receiving Bank Name: The Bank of New York Mellon
+Receiving Bank Address: 240 Greenwich Street, New York NY 10017
+Beneficiary Name: Galaxy Digital Trading Cayman LLC
+Beneficiary Address: 101 Hudson Street, FL 21 Jersey City, NJ 07302
+Beneficiary Account: 8901637157
+
+6. The director of the Trust Sponsor is instructed to take or enter into all required actions and agreements
+with third parties solely to facilitate the implementation of the New Instructions.
+```
+
+### Whistleblower Bounty
+
+*Authorization*: [Forum Post](https://forum.makerdao.com/t/ads-derecognition-due-to-operational-security-breach/22532)
+
+If this executive proposal passes, the following whistleblower bounty will be paid to forum user Venice Tree as per [The Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security):
```

Comment:
I believe this Atlas URL should also be included in the Authorization list

Reply:
The Atlas is not authorizing the payment. The Facilitator (you) is authorizing the payment.

Reply:
It is also stated that the action is clearly taking place "per The Atlas" with the relevant link provided